pkgs/by-name/wf/wfview/package.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  eigen,
+  hidapi,
+  libopus,
+  libpulseaudio,
+  portaudio,
+  qt6,
+  qt6Packages,
+  rtaudio,
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "wfview";
+  version = "2.10";
+
+  src = fetchFromGitLab {
+    owner = "eliggett";
+    repo = "wfview";
+    rev = "v${finalAttr.version}";
+    hash = "sha256-bFTblsDtFAakbSJfSfKgvoxd1DTSv++rxU6R3/uWo+4=";
+  };
+
+  patches = [
+    # Remove syscalls during build to make it reproducible
+    # We also need to adjust some header paths for darwin
+    ./remove-hard-encodings.patch
+  ];
+
+  buildInputs = (
+    [
+      eigen
+      hidapi
+      libopus
+      portaudio
+      rtaudio
+      qt6Packages.qcustomplot
+    ]
+    ++ (with qt6; [
+      qtbase
+      qtserialport
+      qtmultimedia
+      qtwebsockets
+    ])
+    ++ lib.optionals stdenv.hostPlatform.isLinux [
```

```suggestion
      qt6Packages.qcustomplot
      qt6.qtbase
      qt6.qtserialport
      qt6.qtmultimedia
      qt6.qtwebsockets
    ]
    ++ lib.optionals stdenv.hostPlatform.isLinux [
```

It's a bit excessive to add `with` here, also the parentheses wrapping `buildInputs` are unnecessary.